.github/workflows/integration.yml

```diff
@@ -99,11 +105,11 @@ jobs:
         uses: jwalton/gh-docker-logs@v1
         with:
           images: 'ethereumoptimism/builder,ethereumoptimism/hardhat,ethereumoptimism/deployer,ethereumoptimism/data-transport-layer,ethereumoptimism/l2geth,ethereumoptimism/message-relayer,ethereumoptimism/batch-submitter,ethereumoptimism/l2geth,ethereumoptimism/integration-tests'
-          dest: './logs'
+          dest: '~/logs'
```

Comment:
I did this because it was easier to reliably stuff things into `~/logs` from other scripts (namely from `stats.sh`).